tools/engine_tool/lib/src/commands/test_command.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:path/path.dart' as p;
+
+import '../build_utils.dart';
+import '../gn_utils.dart';
+import '../proc_utils.dart';
+import '../worker_pool.dart';
+import 'command.dart';
+import 'flags.dart';
+
+/// The root 'test' command.
+final class TestCommand extends CommandBase {
+  /// Constructs the 'build' command.
+  TestCommand({
+    required super.environment,
+    required Map<String, BuilderConfig> configs,
+  }) {
+    builds = runnableBuilds(environment, configs);
+    debugCheckBuilds(builds);
+    argParser.addOption(
+      configFlag,
+      abbr: 'c',
+      defaultsTo: 'host_debug',
+      help: 'Specify the build config to use',
+      allowed: <String>[
+        for (final Build config in runnableBuilds(environment, configs))
+          config.name,
+      ],
+      allowedHelp: <String, String>{
+        for (final Build config in runnableBuilds(environment, configs))
+          config.name: config.gn.join(' '),
+      },
+    );
+  }
+
+  /// List of compatible builds.
+  late final List<Build> builds;
+
+  @override
+  String get name => 'test';
+
+  @override
+  String get description => 'Runs a test target';
```

Comment:
Ideally this command's help text would help folks learn the selector syntax.

Can a description be multiple lines? It might be useful to add some examples here:

```
et test //flutter/fml/...             # Run all test targets in `//flutter/fml/`
et test //flutter/fml:fml_benchmarks  # Run a single test target in `//flutter/fml/`
```

Reply:
Done.